docs/topics/logging.txt

```diff
@@ -751,5 +751,8 @@ parents, up to the root ``django`` logger. The ``console`` and ``mail_admins``
 handlers are attached to the root logger to provide the behavior described
 above.
 
+For reference, the default logging configuration is defined in
+:source:`django/utils/log.py`.
+
```

Comment:
I would add this to the last sentence, i.e.
```diff
diff --git a/docs/topics/logging.txt b/docs/topics/logging.txt
index c6aa0c5a5a..ca23609cee 100644
--- a/docs/topics/logging.txt
+++ b/docs/topics/logging.txt
@@ -752,4 +752,5 @@ handlers are attached to the root logger to provide the behavior described
 above.
 
 See also :ref:`Configuring logging <configuring-logging>` to learn how you can
-complement or replace this default logging configuration.
+complement or replace this default logging configuration defined in
+:source:`django/utils/log.py`.
```

Reply:
Better.